spec/spec_helper.rb

```diff
@@ -110,4 +110,11 @@ def base_url
 
 def auth_token
   ""
-end
\ No newline at end of file
+end
+
+def standard_headers
+  {
+    "Accept" => "application/json",
+    "Authorization" => "Bearer #{auth_token}",
+  }
+end
```

Comment:
This seemed like it was used in almost every test file, so figured I'd extract a method which would be an easy future cleanup task for someone to use across the codebase.